src/snappy.c

```diff
@@ -126,7 +126,55 @@ static inline bool is_little_endian(void)
 	return false;
 }
 
-#ifndef _MSC_VER
+#if defined(_AIX)
+#define rd_clz(n)   __cntlz4(n)
+#define rd_ctz(n)   __cnttz4(n)
+#define rd_ctz64(n) __cnttz8(n)
+
+#elif defined(sun)
```

Comment:
I believe sun is defined by gcc too and gcc should be providing proper builtins for these things.